components/pipeline-service/production/base/main-pipeline-service-configuration.yaml

```diff
@@ -1576,10 +1576,10 @@ spec:
                   - name: tekton-pipelines-controller
                     resources:
                       requests:
-                        memory: 6Gi
+                        memory: 10Gi
                         cpu: "1"
                       limits:
-                        memory: 8Gi
+                        memory: 12Gi
```

Comment:
Let's start using new pattern of memory request=limits. The reason is  that the pod will be allowed to go higher than requests but will be considered in eviction process if node is under pressure.

I suggest to make both requests and limits 12Gi, WDYT?

Reply:
Yeah, I was aiming at smallest possible change, but you are right. When if not now? :-)